zbus/src/connection/builder.rs

```diff
@@ -77,11 +77,19 @@ assert_impl_all!(Builder<'_>: Send, Sync, Unpin);
 
 impl<'a> Builder<'a> {
     /// Create a builder for the session/user message bus connection.
+    ///
+    /// # Errors
+    ///
+    /// Returns an [`Err`](Result::Err) variant if parsing the session bus address fails.
```

Comment:
Now I understand what you actually meant by "variant" here and why you disagreed with my comment. I thought you mean the "zbus::Error" variant since Result is obvious to even new Rust devs.

My suggest was to either mention the exact `zbus::Error` variant that can be returned from here or just say "error". AFAIK this is the convention too and is useful.

Reply:
I think that's more useful. Updated.